cmd/zoekt-sourcegraph-indexserver/json_schemas/update.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# This script updates the JSON schemas in this directory by cloning the
+# relevant protos from Google and gRPC, and then running protoc-gen-jsonschema
+# on them.
```

Comment:
I don't really understand what these JSON schemas are for. I can't find where we actually use them.

Reply:
These are to help with writing the json for the gRPC service configuration (this way, your editor can help you with auto complete and to see if your schema is valid).

Reply:
I added a unit test that ensures that the default configuration follows this schema.